src/exojax/utils/photometry.py

```diff
@@ -0,0 +1,145 @@
+import numpy as np
+import jax.numpy as jnp
+from exojax.spec.unitconvert import wav2nu
+from exojax.utils.constants import ccgs
+from exojax.utils.url import url_svo_filter
+
+
+def apparent_magnitude(
+    flux_filter, nu_grid_filter, transmission_filter, f0_nu_cgs, factor=1.0e20
+):
+    """computes apparent magnitude
+
+    Args:
+        flux (array): flux in the unit of erg/s/cm^2/cm-1
+        nu_grid_filter (array): wavenumber grid (cm-1)
+        transmission_filter (array): transmission filter (dimensionless, 0 to 1)
+        f0_nu_cgs (float): zero magnitude flux in the unit of erg/s/cm^2/cm-1
+        factor (float): factor to prevent numerical error. Defaults to 1.0e20.
+    """
+
+    logfactor = jnp.log10(factor)
+    integrated_flux = jnp.trapezoid(
+        (flux_filter * factor) * transmission_filter, nu_grid_filter
+    ) / jnp.trapezoid(transmission_filter, nu_grid_filter)
```

Comment:
To reflect the deprecation of jnp.trapezoid 
```suggestion
    integrated_flux = trapezoid(
        (flux_filter * factor) * transmission_filter, nu_grid_filter
    ) / trapezoid(transmission_filter, nu_grid_filter)
```

Reply:
but maybe it's too new?

Reply:
Thank you for your reply! At least in my environment, I encountered the following error when using ``jnp.trapezoid`` with JAX==0.4.16.  So, I followed [this implementation](https://github.com/HajimeKawahara/exojax/pull/476), which was also recommended by [JAX](https://docs.jax.dev/en/latest/changelog.html#jax-0-4-16-sept-18-2023) as you cited. Given that ``jnp.trapz`` was just deprecated in JAX==0.4.16, ``jnp.trapezoid`` is the older version of `jnp.trapz``?

```jax.errors.SimplifiedTraceback: For simplicity, JAX has removed its internal frames from the traceback of the following exception. Set JAX_TRACEBACK_FILTERING=off to include these.

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/moo/exojax/examples/spec_with_photo.py", line 368, in <module>
    Kmag_model = fphoto(
  File "/home/moo/exojax/examples/spec_with_photo.py", line 328, in fphoto
    mag = sop_photo.apparent_magnitude(Fobs)
  File "/home/moo/anaconda3/lib/python3.9/site-packages/ExoJAX-2.0-py3.9.egg/exojax/spec/specop.py", line 161, in apparent_magnitude
    return apparent_magnitude(
  File "/home/moo/anaconda3/lib/python3.9/site-packages/ExoJAX-2.0-py3.9.egg/exojax/utils/photometry.py", line 23, in apparent_magnitude
    integrated_flux = jnp.trapezoid(
  File "/home/moo/anaconda3/lib/python3.9/site-packages/jax/_src/deprecations.py", line 53, in getattr
    raise AttributeError(f"module {module!r} has no attribute {name!r}")
AttributeError: module 'jax.numpy' has no attribute 'trapezoid'```